Assets/Scripts/Capabilities/Dash.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dash : MonoBehaviour
+{
+    [SerializeField] InputController input = null;
+    [SerializeField] MovementOverride movement;
+    [SerializeField] Rigidbody2D rb;
+    [SerializeField] PlayerMovementController controller;
+    
+    private bool isDashing = false;
+    private Vector3 dashDirection;
+    private float dashSpeed;
+
+
+
+    void Update()
+    {
+        if (input.RetrieveDashInput())
+        {
+            isDashing = true;
+            dashDirection = new Vector3(1,0,0);
+            if(!controller.IsFacingRight())
+            {
+                dashDirection = -dashDirection;
+            }
+            gameObject.layer = LayerMask.NameToLayer("Invincible");
```

Comment:
Could we use a constant instead of a string value here? See [constants file](https://github.com/MitchellCoon/MetroidvaniaGameJamFeb23/blob/development/Assets/Scripts/Utils/Constants.cs)

Having all the constants in one centralized place makes development easier, and leads to less bugs where something is slightly misspelled. And if for some reason we ever want to change a layer or tag name, that change only needs to happen in a single place.

Reply:
Good call